app/src/main/AndroidManifest.xml

```diff
@@ -183,14 +172,6 @@
     <service android:name=".webserver.wifi_hotspot.HotspotService" />
 
     <receiver android:name=".main.KiwixSearchWidget">
-      <intent-filter>
```

Comment:
Looking at the [docs](https://developer.android.com/guide/topics/appwidgets#AppWidgetProvider) it seems like we need this and don't need an intent filter on any activity